### PR TITLE
Feat/admin career crud

### PR DESCRIPTION
**Pull request by baron-trieu-goldenowl**
fix api post  -  add url_image post
add crud career
add image career
add WYSIWYG editor to blog

note:
if user add requiment , use edit on OL
image

### CODE REVIEW

File: app/controllers/careers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+  def show
+    respond_to do |format|
+      format.html
+      format.json { render json: @career }
+    end
+  end
+
+  def create
+    @career = Career.create(career_params)
```
Comment (techleadgo) [inline app/controllers/careers_controller.rb:28]: Why did you use `@career = Career.create(career_params)` then use `@career.save`?

File: app/helpers/careers_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module CareersHelper
```
Comment (techleadgo) [inline app/helpers/careers_helper.rb:1]: Remove unused file please

File: app/assets/stylesheets/careers.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Careers controller here.
```
Comment (techleadgo) [inline app/assets/stylesheets/careers.scss:1]: Remove unused file please

File: db/migrate/20200506080706_create_careers.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateCareers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :careers do |t|
+      t.string :title, null: false
+      t.text :content, null: false
+      t.string :jobtype, null: false
```
Comment (techleadgo) [inline db/migrate/20200506080706_create_careers.rb:6]: jobtype or job_type?

File: db/migrate/20200506080706_create_careers.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+      t.string :status, null: false
```
Comment (techleadgo) [inline db/migrate/20200506080706_create_careers.rb:7]: Some columns are not neccessary to add `null: false` (think of it as you are a non-technical user, do you really need to fill `job_type` and `status`?)

File: spec/helpers/careers_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```
Comment (techleadgo) [inline spec/helpers/careers_helper_spec.rb:1]: remove unused file

File: spec/models/career_spec.rb [hunk 7]
```diff
@@ -0,0 +1,18 @@
+# == Schema Information
```
Comment (techleadgo) [inline spec/models/career_spec.rb:1]: Add rspec for this model

File: app/controllers/careers_controller.rb (hunk 1 above)
Comment (baron-trieu-goldenowl) [reply app/controllers/careers_controller.rb:28]: Use @ career.create then check in if the user has submitted it or not

Comment (baron-trieu-goldenowl) [reply app/controllers/careers_controller.rb:28]: Use @ Career.create to create a new career then check if career it has been created

Comment (techleadgo) [reply app/controllers/careers_controller.rb:28]: Use `Model.new` and `Model.save` if you want to check object is saved or not.
`Model.create` does not return true/false

**Pull request merged**